Sources/BraveWallet/Crypto/Asset Details/AssetDetailView.swift

```diff
@@ -36,173 +36,226 @@ struct AssetDetailView: View {
 
   @Environment(\.openURL) private var openWalletURL
   @ObservedObject private var isShowingBalances = Preferences.Wallet.isShowingBalances
-
-  @ViewBuilder private var accountsBalanceView: some View {
-    Section(
-      header: WalletListHeaderView(title: Text(Strings.Wallet.accountsPageTitle)),
-      footer: Button(action: {
-        isShowingAddAccount = true
-      }) {
-        Text(Strings.Wallet.addAccountTitle)
-      }
-        .listRowInsets(.zero)
-        .buttonStyle(BraveOutlineButtonStyle(size: .small))
-        .padding(.vertical, 8)
-    ) {
-      Group {
+  
+  @State private var selectedContent: AssetDetailSegmentedControl.Item = .accounts
+  /// Query displayed in the search bar above the transactions.
+  @State private var query: String = ""
+  @State private var isDoNotShowCheckboxChecked: Bool = false
+  
+  private var accountsBalanceHeader: some View {
+    VStack(spacing: 8) {
+      HStack {
+        Text(Strings.Wallet.accountsPageTitle)
+          .foregroundColor(Color(braveSystemName: .textPrimary))
+          .font(.title3.weight(.semibold))
+          .frame(maxWidth: .infinity, alignment: .leading)
+        
+        Spacer()
+        
         if assetDetailStore.accounts.isEmpty {
```

Comment:
Noticing that `Total balance` can be seen briefly before loading shimmer when first opened. Any easy way to know if we are still fetching accounts vs none found (ex. Filecoin / legacy Solana might not have accounts).

Reply:
will hide the header when there is no account. `isLoadingAccountBalances` will be used to determine when to start shimmering.